MaxText/layers/linears.py

```diff
@@ -156,9 +247,13 @@ def compute_dot_general(inputs, kernel, axis, contract_ind):
 
     contract_ind = tuple(range(0, len(axis)))
     output = compute_dot_general(inputs, kernel, axis, contract_ind)
+    pre_bias_logits = None
 
     if self.score_func:
       output = _convert_to_activation_function(self.score_func)(output)
+      if self.decoder_block == "deepseek" and self.use_bias:
```

Comment:
Could you check deepseek v3? e.g. config.model_name.startswith("deepseek3")

Reply:
sounds good, will update